docs/internals/howto-release-django.txt

```diff
@@ -214,13 +257,10 @@ any time leading up to the actual release:
     $ git checkout -b stable/4.2.x origin/stable/4.1.x
     $ git push origin stable/4.2.x:stable/4.2.x
 
-Preparing for release
```

Comment:
I think this section title was adding confusion since the steps before this are also "preparing for release".